defender/tests/Autotask_Forta_Explorer_Monitor.spec.js

```diff
@@ -0,0 +1,393 @@
+jest.mock('axios', () => jest.fn());
+const axios = require('axios');
+
+// this will allow us to override values returned by Date Class methods
+jest.useFakeTimers();
+jest.setSystemTime(8675309);
+
+const mockKeyValueStore = {
+  get: jest.fn(),
+  put: jest.fn(),
+};
+
+const botIds = [
+  '0x5a00b44b2db933d4c797e6bd3049abdeb89cc9ec1b2eaee7bdbaff911794f714', // Forta Low Liquidity Attack
+  '0xb6bdedbae67cc82e60aad02a8ffab3ccbefeaa876ca7e4f291c07c798a95e339', // Forta Large Borrows Governance
+  '0x916603512086fcad84c35858d2fc5356c512f72b19c80e52e8f9c04d8122e2ba', // Forta Multi-Sig Monitor
+  '0x0d3cdcc2757cd7837e3b302a9889c854044a80835562dc8060d7c163fbb69d53', // Forta Large Delegations Monitor
+  '0xe200d890a67d51c3610520dd9fdfa9e2bd6dd341d41e32fa457601e73c4c6685', // Forta Oracle Price Monitor
+  '0xf836bda7810aa2dd9df5bb7ac748f173b945863e922a15bb7c57da7b0e6dab05', // Forta Underlying Asset Monitor
+];
+
+// override the key-value store Class
+jest.mock('defender-kvstore-client', () => ({
+  KeyValueStoreClient: jest.fn().mockReturnValue(mockKeyValueStore),
+}));
+
```

Comment:
```suggestion
    // set up a capture of any put() calls to the kvstore
    const outputObject = {};
    mockKeyValueStore.put.mockImplementation((inputKey, value) => {
      outputObject[inputKey] = value;
    });
```
NIT: If the same implementation is used everywhere, you can just set it at the global scope and remove it from the tests. Just a light reduction of code. You can leave it as is if you prefer the explicit approach.